bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -197,53 +379,72 @@ public void receiveCommand(String itemName, Command command, BticinoBindingConfi
                 }
                 m_open_web_net.onCommand(l_pr);
             }
-        } catch (Exception e) {
-            logger.error("Gateway [" + m_gateway_id + "], Error processing receiveCommand '{}'",
-                    (Object[]) new String[] { e.getMessage() });
+        } catch (
+
+        Exception e) {
+            logger.error("Gateway [{}], Error processing receiveCommand '{}'", m_gateway_id,
```

Comment:
This should be a WARN, not an ERROR.

Reply:
OK
See commit https://github.com/openhab/openhab1-addons/pull/5486/commits/477b974d34637cd0f61a6b69fb77755f0d094913